File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    dic = dict()
    for v,k in clothes:
        if k in dic:
            dic[k] += 1
        else:
            dic[k] = 1
    for n in dic.values():
        answer*=(n+1)
    return answer-1

File: 프로그래머스/2/42578. 의상/의상.cpp
#include <string>
#include <vector>
#include <map>

using namespace std;

int solution(vector<vector<string>> clothes) {
    int answer = 1;
    map<string, int> m;
    for(int i = 0; i < clothes.size(); i++){
        m[clothes[i][1]] += 1;
    }
    for(auto& [type, count] : m){
        answer = answer * (count+1);
    }
    return answer-1;
}